webapp/src/components/conference/conference.tsx

```diff
@@ -232,7 +241,7 @@ export default class Conference extends React.PureComponent<Props, State> {
 
         return (
             <div style={style.buttons}>
-                {this.state.minimized && this.state.position === POSITION_TOP &&
+                {!this.props.showPrejoinPage && this.state.minimized && this.state.position === POSITION_TOP &&
```

Comment:
@larkox , there is another outstanding Jitsi ticket to enable user-resizing of the Jitsi window that the product team was going to review/prioritize. For the time being, Katie agreed that we should provide a taller window so the pre-join page isn't clipped. For users that disable the prejoin, we decided to leave the window as-is since it isn't clipped.